### ISSUE TITLE
Nemo crash on moving last file of a folder

### ISSUE THREAD

**Issue by samilouahala** [linuxmint/nemo#2170]
```
 * Nemo 4.0.6
 * windowed nemo
 * Mint 19.1
 * asus GTX780  Nvidia driver 418.67
 * 64 bit
 ```

**Issue**
Almost every time I move (drag-drop) the last file of a folder to another folder in the same window, all window instances of Nemo close down.

**Steps to reproduce**

Moving last file of a folder
[_usr_bin_nemo.1000.crash.tar.gz](https://github.com/linuxmint/nemo/files/3380765/_usr_bin_nemo.1000.crash.tar.gz)

**Expected behaviour**

All instances of Nemo stay open

**Other information**

**Comment by clefebvre** [comment]
Can you provide a stack trace and the hierarchy of files/folders being moved?

Try to reproduce the issue with 4.2 also if possible from the Alpha testing PPA: https://linuxmint-developer-guide.readthedocs.io/en/latest/daily_builds_ppa.html

**Comment by mtwebster** [comment]
Looking at trace, I think this is a duplicate of https://github.com/linuxmint/nemo/issues/2150 (fixed for 4.2)

**Comment by dannn-o** [comment]
I encountered this issue with one of the many "Issues" closed as a suspected duplicate of #2150 , and since this is one of the few remaining open, I wanted to ask the question here:  What is the easiest way to incorporate this particular fix, if at all possible without having to resort to installing a GIT clone (if that's the correct terminology) or using the Alpha testing (or any other) PPA?  Is version 4.2 (which will apparently include the fix) going to be incorporated when the upgrade path to Linux Mint 19.2 becomes available?  Thanks.

**Comment by JosephMcc** [comment]
@dannn-o Cinnamon and Nemo 4.2 will be part of Mint 19.2 so you will get it by upgrading when it's available.

**Comment by dannn-o** [comment]
Thanks.  But, how does one confirm that a particular fix is contained with a given version of Nemo?  Will Mint 19.2 be the earliest that the fix / v4.2 would be available, or might it be released in an update with Mint 19.1?
